tests/unit/test_set_integrations_env.py

```
@@ -74,6 +74,38 @@ def get(self, *a, **k):
 
+@mock.patch(
+    "deepnote_toolkit.set_integrations_env.get_config",
+    side_effect=lambda: _mock_cfg(True),
+)
+def test_set_integration_env_clears_config_cache(
+    mock_get_config, monkeypatch  # noqa: ARG001
+):
+    class DummyResp:
+        ok = True
+
+        def json(self):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `json`

(ANN202)